src/main/thrift/parquet.thrift

```diff
@@ -1041,6 +1041,13 @@ struct FileMetaData {
    * Used only in encrypted files with plaintext footer. 
    */ 
   9: optional binary footer_signing_key_metadata
+
+  /**
+   * This field might be set with the version number of a parquet-format release
+   * if this file is created by using only the features listed in the related
+   * list of core features. See CoreFeatures.md for details.
```

Comment:
It might be useful to clarify how this differs from the `version` key above ?

Reply:
It's a good point, @jorisvandenbossche. However, I am not sure what the other version is for. Maybe, it would be better to specify that one correctly and it would be clear that this one is different thing.
Currently parquet-mr always writes `1` to that version field. I guess this is a file format version that we never incremented as it is backward compatible to the first ones we wrote. If it is the case then I think we will never increment it.

Reply:
In Arrow (parquet-cpp) this `version` field actually gets populated with `1` or `2`, and depending on that different features are used to write the file (version 2 enabled some additional features, like writing nanosecond timestamps, or using RLE_DICTIONARY instead of PLAIN_DICTIONARY)

Reply:
It doesn't sound good. It means that we are using a required field for different purposes. Because it is required we cannot deprecate it easily.
parquet-mr does not read this field only writes `1` all the time. What does parquet-cpp does with this value at the read path?

Reply:
AFAIK, the field is not used when reading (but I am not super familiar with the C++ code)

Reply:
FWIW, Impala reads this field and fails if it is 2 (https://issues.apache.org/jira/browse/IMPALA-11838?filter=-2)